cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrap.scala

```diff
@@ -120,10 +120,15 @@ object ClusterBootstrap extends ExtensionId[ClusterBootstrap] with ExtensionIdPr
 
   override def createExtension(system: ExtendedActorSystem): ClusterBootstrap = new ClusterBootstrap()(system)
 
-  private[bootstrap] sealed trait BootstrapStep
-  private[bootstrap] case object NotRunning extends BootstrapStep
-  private[bootstrap] case object Initializing extends BootstrapStep
-  // TODO get the Initialized state once done
-  private[bootstrap] case object Initialized extends BootstrapStep
+  /**
+   * INTERNAL API
+   */
+  private[bootstrap] object Internal {
+    sealed trait BootstrapStep
+    case object NotRunning extends BootstrapStep
+    case object Initializing extends BootstrapStep
+    // TODO get the Initialized state once done
```

Comment:
Shall we just remove the Initialized? Not used and ClusterBootstrap sends a fire and forget start rather than an ask

Reply:
ok